### PR TITLE
add cta demo form popup to SAAS sign up workflow

### PR DESCRIPTION
**Pull request by biyeun**
@gbova cc @orangejenny

### CODE REVIEW

**Review by nickpell — APPROVED** [review]

File: corehq/apps/hqwebapp/templates/hqwebapp/base.html [hunk 1]
```diff
@@ -547,6 +550,37 @@
           <script src="{% static 'hqwebapp/js/validators.ko.js' %}"></script>
         {% endif %}
 
+        {% if not equest.user.is_authenticated and is_saas_environment %}
```
Comment (orangejenny) [inline corehq/apps/hqwebapp/templates/hqwebapp/base.html]: `request`

File: corehq/apps/hqwebapp/templates/hqwebapp/base.html [hunk 2]
```diff
@@ -547,6 +550,37 @@
...
+              </div>
+            </div>
+          </div>
+          <!--[if lte IE 8]>
+          <script charset="utf-8" type="text/javascript" src="//js.hsforms.net/forms/v2-legacy.js"></script>
+          <![endif]-->
+          <script charset="utf-8" type="text/javascript" src="//js.hsforms.net/forms/v2.js"></script>
+          <script>
+            hbspt.forms.create({
+              portalId: "503070",
```
Comment (orangejenny) [inline corehq/apps/hqwebapp/templates/hqwebapp/base.html]: Should pull from `ANALYTICS_IDS.HUBSPOT_API_ID`

File: corehq/apps/hqwebapp/templates/hqwebapp/base.html [hunk 3]
```diff
@@ -547,6 +550,37 @@
...
+              formId: "0f5de42e-b562-4ece-85e5-cfd2db97eba8",
```
Comment (orangejenny) [inline corehq/apps/hqwebapp/templates/hqwebapp/base.html]: Don't we usually store these server-side?

File: corehq/apps/hqwebapp/static/hqwebapp/less/_hq/forms.less [hunk 4]
```diff
@@ -462,3 +462,81 @@ table .ko-inline-edit .read-only:hover {
 .radio-select_label-placeholder {
   height: @input-height-base;
 }
+
+
+.hs-input {
```
Comment (orangejenny) [inline corehq/apps/hqwebapp/static/hqwebapp/less/_hq/forms.less]: `hubspot` would be better than `hs`

Comment (orangejenny) [reply corehq/apps/hqwebapp/static/hqwebapp/less/_hq/forms.less]: (not blocking)

File: corehq/apps/hqwebapp/templates/hqwebapp/base.html (hunk 3 above)
Comment (orangejenny) [reply corehq/apps/hqwebapp/templates/hqwebapp/base.html]: (not blocking)

**Review by orangejenny — CHANGES_REQUESTED** [review]
two small changes, see comments

File: corehq/apps/hqwebapp/templates/hqwebapp/base.html (hunk 1 above)
Comment (biyeun) [reply corehq/apps/hqwebapp/templates/hqwebapp/base.html]: hmmm how did that sneak in there

File: corehq/apps/hqwebapp/templates/hqwebapp/base.html (hunk 3 above)
Comment (biyeun) [reply corehq/apps/hqwebapp/templates/hqwebapp/base.html]: it's on a public page anyway?

File: corehq/apps/hqwebapp/static/hqwebapp/less/_hq/forms.less (hunk 4 above)
Comment (biyeun) [reply corehq/apps/hqwebapp/static/hqwebapp/less/_hq/forms.less]: it's the style that hubspot inserts. I can't change it unfortunately.

**Comment by orangejenny** [comment]
it's the `API_ID` not `API_KEY`, feel free to self-merge after that

**Review by orangejenny — APPROVED** [review]

**Pull request merged**